### DISCUSSION TITLE
关于 Loading 加载开发相关

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#42] [General]
## 说明

` Loading` 加载组件用于数据加载时显示的动效。

## 开发者

该组件由 [yn22638](https://github.com/yn22638) 进行开发

## 设计稿

https://www.figma.com/file/ihsP3fTNCvqrHG1tV0R3sQ/Loading-%E5%8A%A0%E8%BD%BD?node-id=0%3A1

**Comment by yn22638** [comment]
textColor ：字体颜色，overlayBgColor:遮罩层背景，loadingText:自定义加载内容，show：是否显示 loading, duration:动画时长
